PhysicsTools/PatAlgos/test/corMETFromMiniAOD.py

```diff
@@ -45,17 +45,17 @@
   process.GlobalTag.globaltag = autoCond['run2_data']
   #process.GlobalTag.globaltag = '75X_dataRun1_v2' #'74X_dataRun2_Prompt_v1'
 else:
-  process.GlobalTag.globaltag = 'MCRUN2_75_V5'
- # process.GlobalTag.globaltag = autoCond['run2_mc']
+ # process.GlobalTag.globaltag = 'MCRUN2_75_V5'
```

Comment:
does it make sense to keep this old global tag here?
Does it at least correspond to some large production? (75_ part suggests that it does not)